consequence_prediction/repeat_expansion_variants/pipeline.py

```diff
@@ -82,8 +89,13 @@ def load_clinvar_data(clinvar_xml):
 def parse_variant_identifier(row):
     """Parse variant identifier and extract certain characteristics into separate columns."""
     variant_name = str(row.Name)
-    row['TranscriptID'], row['CoordinateSpan'], row['RepeatUnitLength'], row['IsProteinHGVS'] = \
+    transcript_id, coordinate_span, repeat_unit_length, is_protein_hgvs = \
         none_to_nan(*clinvar_identifier_parsing.parse_variant_identifier(variant_name))
+    row['TranscriptID'], row['RepeatUnitLength'], row['IsProteinHGVS'] = \
+        transcript_id, repeat_unit_length, is_protein_hgvs
+    # Only overwrite the coordinate span inferred previously if we got something better by parsing the identifier.
+    if not np.isnan(coordinate_span):
+        row['CoordinateSpan'] = coordinate_span
```

Comment:
In this case we would be preferring the length of insertion given within the identifier than the one we inferred with the ``ALT`` and ``REF`` alleles, correct? 

Could you give me an example of the format of the former? In other words, what would be an identifier from which you could infer the length without requiring the allele lengths.

Reply:
That's correct, here's an example (not one of the 7 nameless ones, but it is a repeat expansion): `NC_000020.11:g.2652734_2652739[(16_?)]`.  In this case there actually are REF and ALT given as part of the `SequenceLocation` element, but in general they're not always present.

I could see an argument for preferring the coordinate span computed from the alleles if it's provided, but this would be a more significant change since previously we were always parsing coordinate span from the identifier, and I feel like I haven't fully thought through the impact... maybe I can test it out and see...

Reply:
Don't we do it anyway if the identifier does not contain the ``start`` and ``end`` coordinates? I think the way it is done currently is fine: inferring is usually worse than taking valid information from the identifier. 

I was just curious about the format of the given coordinates in this case.